td-models/api.yml

```diff
@@ -238,54 +319,57 @@ components:
         id:
           type: string
           description: A Causemos model UUID
-        statements:
+        nodes: 
+          type: array
+          description: Nodes and their parameterizations
+          items:
+            oneOf:
+              - $ref: "#/components/schemas/NodeParameter"
+        edges:
           type: array
-          description: Causemos INDRA statement
           items:
             type: object
-        conceptIndicators:
-          type: object
-          description: Node parameters
-          properties:
-            <conceptName>:
-              type: object
-              description: Concept node name
-              properties:
-                name:
-                  type: string
-                func:
-                  type: string
-                numLevels:
+            properties:
+              source:
+                type: string
+                description: source concept
+              target:
+                type: string
+                description: target concept
+              polarity:
+                type: number
+                description: edge polarity in Causemos. 1=same, -1=opposite, unknown/ambiguous otherwise
+              weights:
+                type: array
+                description: denotes overriding weights analyts want to impose, for DySE this will be [level, trend], for Delphi this will be [weight]
```

Comment:
```suggestion
                description: denotes overriding weights analysts want to impose, for DySE this will be [level, trend], for Delphi this will be [weight]
```